tests/run_tests.sh

```diff
@@ -130,7 +130,12 @@ function setup_test_options()
     # for the scenario of specifying test scripts using pattern like `subfolder/test_*.py`. The pattern will be
     # expanded to matched test scripts by bash. Among the expanded scripts, we may want to skip a few. Then we can
     # explicitly specify the script to be skipped.
-    ignores=$(python -c "print '|'.join('''$SKIP_FOLDERS'''.split())")
+    python_version="$(python -V 2>&1)"
+    if [[ $python_version =~ "Python 3" ]]; then
+        ignores=$(python -c "print('|'.join('''$SKIP_FOLDERS'''.split()))")
+    else
+        ignores=$(python -c "print '|'.join('''$SKIP_FOLDERS'''.split())")
```

Comment:
I think python 2 also supports command like `python -c "print('hello world')"`. It's unnecessary to apply this logic.